app/mailers/user_mailer.rb

```diff
@@ -9,6 +9,12 @@ def authentication(authentication)
     mail to: @authentication.user.email_address, subject: "Sign in to Hackathons"
   end
 
+  def hackathon_submission
+    @hackathon_name = params[:name]
+    @admin_email = params[:admin_email]
+    mail to: @admin_email, subject: 'A new Hackathon entry was submitted!'
+  end
+
```

Comment:
I don't love putting this in the `UserMailer`. Maybe we could have a `HackathonMailer` that provides a `submission`?

Reply:
I will refactor this, thanks for the suggestion!